docs/docusaurus/docs/guides/connecting_to_your_data/components/where_to_run_code.md

```diff
@@ -6,23 +6,13 @@ Get an environment to run the code in this guide. Please choose an option below.
 
 <Tabs
   groupId="yaml-or-python"
-  defaultValue='cli'
+  defaultValue='yaml'
   values={[
-  {label: 'CLI + filesystem', value:'cli'},
   {label: 'No CLI + filesystem', value:'yaml'},
   {label: 'No CLI + no filesystem', value:'python'},
```

Comment:
I think we should update these labels.  Maybe something like: "file editing" and "python" -- or even remove the "file editing" option, since I believe we're trying to get people away from directly editing yaml.